core/src/main/java/io/github/tcdl/Producer.java

```diff
@@ -1,58 +1,61 @@
-package io.github.tcdl;
```

Comment:
Could you comment what actually changed in the class in such cases when due to difference in formatting the file looks completely different?